services/ingestion/subscriber.go

```diff
@@ -146,7 +146,16 @@ func (r *RPCSubscriber) subscribe(ctx context.Context, height uint64, opts ...ac
 					return
 				}
 
-				events <- models.NewBlockEvents(blockEvents)
+				evts := models.NewBlockEvents(blockEvents)
+				if evts.Err != nil {
+					r.logger.Warn().Err(err).Msgf(
+						"failed to parse EVM block events for Flow height: %d, retrying with gRPC API...",
+						blockEvents.Height,
+					)
+					events <- r.fetchBlockEvents(ctx, blockEvents)
```

Comment:
add a commnet something like:

```
// call an alternative grpc request to fetch events in case the event streaming failed
```

Reply:
Added in https://github.com/onflow/flow-evm-gateway/pull/529/commits/76d544c7e08a528858e034b4e99bf965a4361830